cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sputnikvm-cli"
-version = "0.6.0"
+version = "0.0.0"
```

Comment:
is this supposed to be `0.10.0`?

Reply:
@shanejonas That is meant to "unpublish" `sputnikvm-cli`. We don't distribute binary artifacts for it and currently its interface might change a lot, plus it is currently mostly used internally, so I figured it might be better to just ask everyone who needs that tool to just build from `master`. That's why it is set to `0.0.0`.